src/pcre2_extuni.c

```diff
@@ -40,7 +40,7 @@ POSSIBILITY OF SUCH DAMAGE.
 
 /* This module contains an internal function that is used to match a Unicode
 extended grapheme sequence. It is used by both pcre2_match() and
-pcre2_def_match(). However, it is called only when Unicode support is being
+pcre2_dfa_match(). However, it is called only when Unicode support is being
```

Comment:
is this (the need for the dummy when unicode support is not compiled that will be never called) still true?, it is confusing but it would seem that ECLASS calls this unconditionally.

Reply:
I don't think that the ECLASS code calls into pcre2_extuni at all.

I haven't changed anything in here, except noticed a typo in a comment that was copy-and-pasted into multiple files.